etc/kayobe/seed.yml

```diff
@@ -112,8 +112,8 @@ seed_pulp_container:
     # s6-overlay-suexec starts as pid 1
     init: false
     env:
-      PULP_CONTENT_WORKERS: "{{ ansible_facts.processor_vcpus * 2 + 1 }}"
-      PULP_API_WORKERS: "{{ ansible_facts.processor_vcpus * 2 + 1 }}"
+      PULP_CONTENT_WORKERS: "{{ [ansible_facts.processor_vcpus * 2 + 1, 32] | min }}"
+      PULP_API_WORKERS: "{{ [ansible_facts.processor_vcpus * 2 + 1, 32] | min }}"
```

Comment:
I realise this is just a sync, but this should be `max`, right?

Reply:
The goal is to avoid running too many workers on hosts with many CPU cores, which is why we use `min`:

* on a host with 4 CPU core: `min(4 * 2 + 1, 32) = min(9, 32) = 9`
* on a host with 256 CPU cores: `min(256 * 2 + 1, 32) = min(513, 32) = 32`

Reply:
The original uncapped formula would try running e.g. 513 workers on a 256-core host, which would fail due to resource exhaustion.

Reply:
Of course. Brain not working after a 4am start :(